app/src/main/java/org/mozilla/focus/navigation/MainActivityNavigation.kt

```diff
@@ -137,7 +137,8 @@ class MainActivityNavigation(
      * Show first run onboarding.
      */
     fun firstRun() {
-        val onboardingFragment = if (Features.ONBOARDING) {
+        val onboardingFragment = if (FocusNimbus.features.onboarding.value().isEnabled) {
+            FocusNimbus.features.onboarding.recordExposure()
```

Comment:
This is interesting. Given that `isEnabled` is part of the feature values, I think we'd want to record exposure whatever the value of `isEnabled`.

Additionally, I think the `FeatureHolder` assumes that `value()` is called _before_ `recordExposure()`, so you could do this:

```suggestion
        val isOnboarding = FocusNimbus.features.onboarding.value().isEnabled
        FocusNimbus.features.onboarding.recordExposure()
        val onboardingFragment = if (isOnboarding) {
```

or:

```kotlin
        val onboardingFragment = if (FocusNimbus.features.onboarding.value().isEnabled) {
            FocusNimbus.features.onboarding.recordExposure()
            OnboardingFragment.create()
        } else {
            FocusNimbus.features.onboarding.recordExposure()
            FirstrunFragment.create()
        }   
```